gwt-ol3-client/src/main/java/ol/MapOptions.java

```diff
@@ -32,49 +35,110 @@
 @JsType(isNative = true, namespace = JsPackage.GLOBAL, name = "Object")
 public class MapOptions implements Options {
 
+    /**
+     * @param controls Controls initially added to the map. If not specified, <code>ol.control.defaults()</code> is used.
+     */
     @JsProperty
-    public native double getPixelRatio();
+    public native void setControls(Collection<Control> controls);
 
     @JsProperty
-    public native String getTarget();
+    public native Collection<Control> getControls();
 
+	/**
+	 * @param pixelRatio The ratio between physical pixels and device-independent pixels (dips) on the device.
+	 * If <code>undefined</code> then it gets set by using <code>window.devicePixelRatio</code>.
+	 */
     @JsProperty
-    public native void setTarget(String target);
+    public native void setPixelRatio(Double pixelRatio);
 
     @JsProperty
-    public native void setLayers(Collection<Base> layers);
+    public native Double getPixelRatio();
+
+	/**
+	 * Interactions that are initially added to the map. If not specified,
+	 * ol.interaction.defaults() is used.
+	 *
+	 * @param interactions
+	 */
+	@JsProperty
+	public native void setInteractions(Collection<Interaction> interactions);
 
     @JsProperty
-    public native View getView();
+    public native void setInteractions(Interaction[] interactions);
```

Comment:
[OL documentation](https://openlayers.org/en/latest/apidoc/ol.Map.html) states that method can accept both Collection and array same as for layers property. Setters for layers property were already present, so I just duplicated their logic, allowing passing bot Collection and array.